### DISCUSSION TITLE
How to stream data between two connectors via http?

### DISCUSSION THREAD

**Discussion by dcandas** [eclipse-edc/Connector#960] [Q&A] [answered]
We managed a successful negotiation, but now we are stuck at transferring data from provider to consumer. We want to transfer data using http streams. We have tried the `/transfer` and `/control/transfer` endpoints, which eventually require a validation endpoint, which seems to be only available using the dataplane-transfer-sync.

What is the suggested way to stream http data between two connectors?

**Comment by dilip-dhankecha** [comment, marked as answer]
ref: https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/discussions/1096

**Comment by mspiekermann** [comment]
Please have a look at https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/903 and corresponding linked PRs, Issues and comments.
